README.md

```diff
@@ -1,2 +1,14 @@
 # Adapter
 this repo contains Adapter design pattern implementation using C#
+
+## The problem 
+ 
+![UML](Problem.gif)
+
+I have faced this problem in one of my previous projects. And as a result, I ended up solving the problem by using the 'Adapter design pattern'. The way that I did was by adding a class ,Adapter(), that plays as the adapter role.  
```

Comment:
I liked how you reflected what you've learn on your previous projects! That's really great.